classes/Form/ConfigurationForm.php

```diff
@@ -221,6 +239,11 @@ public function treat()
             $treatmentResult .= $this->module->displayConfirmation($this->module->getTranslator()->trans('Account ID updated successfully', [], 'Modules.GAnalytics.Admin'));
         }
 
+        if (null !== $gaV4Enabled) {
+            Configuration::updateValue('GA_V4_ENABLED', (bool) $gaV4Enabled);
+            $treatmentResult .= $this->module->displayConfirmation($this->module->l('Settings for Google Analytics 4 updated successfully'));
```

Comment:
`$this->module->trans`, for new line 259, 264 as well

Reply:
Fixed

Reply:
lgtm :ok_hand: